lib/fennec/auth.ex

```diff
@@ -78,11 +78,12 @@ defmodule Fennec.Auth do
   defp should_authorize?(:request, :allocate), do: true
   defp should_authorize?(:request, :create_permission), do: true
   defp should_authorize?(:request, :refresh), do: true
+  defp should_authorize?(:indication, :send), do: true
```

Comment:
From this excerpt of the TURN RFC it seems that indications should not be authorised at all:

> Send and Data indications cannot be authenticated, since the long-
   term credential mechanism of STUN does not support authenticating
   indications.